src/Common.Lib/Macros.h

```diff
@@ -65,8 +65,8 @@
 #endif
 
 #ifndef IfFalseRet
-#define IfFalseRet(EXPR) \
-    do { if (FALSE == (EXPR)) { CLogging::LogError(_T("IfFalseRet(") _T(#EXPR) _T(") failed in function ") __FUNCTIONT__); return E_FAIL; } } while (false)
+#define IfFalseRet(EXPR, HR) \
```

Comment:
>HR [](start = 25, length = 2)

super nit, but I think it would improve readability of the HR is the first parameter so that it would read "IfFalseRet(E_Fail, .....)"

Reply:
I know this would change a lot of code, so I don't mind it as-is.

---
In reply to: [381498139](https://github.com/microsoft/CLRInstrumentationEngine/pull/204#discussion_r381498139) [](ancestors = 381498139)

Reply:
I think it's more important to read what is being tested first rather than what the return value is for failure. Also, there will be cases where "HRESULT_FROM_WIN32(GetLastError())" is the return value, which would push back the tested value much further out of view.